CHANGELOG.md

```diff
@@ -1285,6 +1285,24 @@ Read the [announcement blogpost here](https://reactnative.dev/blog/2023/01/12/ve
 
 - Encode URL params in URLSearchParams.toString() ([1042a8012f](https://github.com/facebook/react-native/commit/1042a8012fb472bd5c882b469fe507dd6279d562) by [@sshic](https://github.com/sshic))
 
+## v0.68.7
+
+### Fixed
+
+#### iOS
+
+- Use logical operator instead of bit operation in Yoga ([c3ad8](https://github.com/facebook/react-native/commit/c3ad8ec7eb01b7236e0081ac7c7f888630caac21) by [@cuva](https://github.com/cuva))
```

Comment:
Actually, I think tha thtis is valid also for Android, as it is the `Yoga.cpp` implementation file.